### ISSUE TITLE
Wrong calculation of size and remainingLength in method `cloudletResume` in `CloudletSchedulerSpaceShared`

### ISSUE THREAD

**Issue by NhatPhan** [Cloudslab/cloudsim#100]
This piece of codes in `cloudletResume` is buggy:
![image](https://user-images.githubusercontent.com/12751429/29610760-cf838952-882d-11e7-90a4-a7dc9e5c824d.png)

1. in line 257 `rcl.getRemainingCloudletLength()` is assigned to `size`, the `getRemainingCloudletLength` : 
![image](https://user-images.githubusercontent.com/12751429/29611089-d58af35c-882e-11e7-88a6-ff6994beca96.png)
which uses `getCloudletTotalLength`, which already includes `getNumberOfPes()`. however on line 258 `size` is multiplied by `getNumberOfPes()`, making the value wrong.
2. in line 259 the cloudletLength is updated with `size` (which is the remaining cloudlet length). but the `cloudletFinishedSoFar` is not reset to 0. therefore subsequent calls to `getRemainingCloudletLength()` (like line 276) will get the wrong value since the cloudletFinishedSoFar will be subtracted twice.

**Comment by douglasleer** [comment]
I agree with you on these two errors. I actually found many more errors in other places within the code.
